### PR TITLE
hwdb: Add accelerometer orientation quirk for the Teclast F6 Pro

### PR DESCRIPTION
**Pull request by ljmf00**
This pull request add to hwdb, the accelerometer matrix for correct orientation on the display for Teclast F6 Pro

### CODE REVIEW

**Comment by ljmf00** [comment]
@poettering can you see whats wrong?

File: hwdb/60-sensor.hwdb [hunk 1]
```diff
@@ -475,6 +475,10 @@ sensor:modalias:acpi:KIOX000A*:dmi:bvnAmericanMegatrendsInc.:bvr5.011:bd11/03/20
 # Teclast F5
 sensor:modalias:acpi:KIOX010A*:dmi:*:svnTECLAST:pnF5:*
  ACCEL_MOUNT_MATRIX=0, 1, 0; 1, 0, 0; 0, 0, 1
+
```
Comment (yuwata) [inline hwdb/60-sensor.hwdb]: Please drop white space in the blank line.

**Comment by poettering** [comment]
> @poettering can you see whats wrong? CI failed for some reason

As @yuwata mentioned, there's spurious whitespace in your patch. We generally require (as most bigger projects) that our sources are whitespace-clean, i.e. have no trailing whitespace. We ship .editorconfig, .dir-locals.el and .vimrc files in our tree though so that for most editors this should just work, and the editors clean this up implicitly. I am a bit surprised it didn't work for you automatically though...

**Comment by ljmf00** [comment]
sorry for the inconvenient. The situation is already fixed and should be ready to merge now

**Comment by ljmf00** [comment]
@poettering the ci is now reporting something unrelated to this pull request

**Comment by MadhiasM** [comment]
I see the correct mount matrix in 60-sensor.hwdb:
grafik
But despite the fact, I see in iio that the mount matrix is initialized different for both modules:
grafik
This results in the screen orientation and tablet mode not working properly. Any idea why the mount matrix vom 60-sensor.hwdb is not correctly passed through iio-sensor-proxy?

I am running Zorin OS 17.2 Kernel 6.8.0-48 on F6 Pro
grafik

**Pull request merged**